### PR TITLE
Added calls host control functionality to docs

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for:
- https://github.com/mattermost/mattermost-plugin-calls/milestone/34?closed=1
- https://github.com/mattermost/mattermost/pull/27031

### CODE REVIEW

File: source/collaborate/make-calls.rst [hunk 1]
```diff
@@ -50,10 +56,47 @@ Start a call
...
 
+Host controls
+-------------
+
+From Mattermost v9.9, call host controls are available when using Mattermost in a web browser, the desktop app, or the mobile apps, and include the ability to `transfer host duties <#transfer-host-duties>`__, `remove call participants <#remove-call-participants>`__, `stop a screen share <#stop-a-screen-share>`__, `mute or unmute participants <#mute-or-nmute-participants>`__, and `lower raised hands <#lower-raised-hands>`__. 
+
+Access host controls
+~~~~~~~~~~~~~~~~~~~~
+
+Host controls are available to call hosts in both the call widget by selecting the **More** |more-icon| icon next to a participant's name, and in the expanded the call window as hosts hover over a call participant in the list.
```
Comment (cpoile) [inline source/collaborate/make-calls.rst:67]: Maybe:
```suggestion
Host controls are available to call hosts and admins in both the call widget by selecting the **More** |more-icon| icon next to a participant's name, and in the expanded the call window as hosts hover over a call participant in the list.
```

File: source/collaborate/make-calls.rst [hunk 2]
```diff
@@ -50,10 +56,47 @@ Start a call
...
+
+Transfer host duties
+~~~~~~~~~~~~~~~~~~~~
+
+Transfer host duties to another call particpant by accessing the host controls and selecting **Make host**. Once host duties are transfered to someone else, you can't access host controls unless they're transferred back to you.
```
Comment (cpoile) [inline source/collaborate/make-calls.rst:72]:
```suggestion
Transfer host duties to another call participant by accessing the host controls and selecting **Make host**. Once host duties are transferred to someone else, you can't access host controls unless they're transferred back to you. System admins can change the host at any time.
```

File: source/collaborate/make-calls.rst [hunk 3]
```diff
@@ -50,10 +56,47 @@ Start a call
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Remove a call participant from an active call by accessing the host controls and selecting **Remove from call**, then confirm by selecting **Yes, remove**. The call participant is notified that they've been removed from the call by the host.
+
+Stop a screen share
+~~~~~~~~~~~~~~~~~~~~
+
+Stop a call participant's screen share by accessing the host controls and selecting **Stop screen share**.
+
+Mute or unmute participants
```
Comment (cpoile) [inline source/collaborate/make-calls.rst:84]: We didn't implement the unmute participants feature, so:
```suggestion
Mute participants
```

Comment (cpoile) [reply source/collaborate/make-calls.rst:84]: And I suppose the below needs to be changed too :)

File: source/collaborate/make-calls.rst [hunk 4]
```diff
@@ -50,10 +56,47 @@ Start a call
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Invite muted participants to unmute their microphone by accessing the host controls and selecting **Ask to unmute**.  The call particpant is prompted to decide whether unmute or stay muted.
+
+You can mute or unmute the microphone of specific particpants by accessing the host controls and toggling the particpant's mute icon. Mute all call particpants by selecting **Mute all**, and unmute all participants by selecting **Unmute all**.
+
+Lower raised hands
+~~~~~~~~~~~~~~~~~~~
+
+Lower a raised hand by accessing the host controls and selecting **Lower hand**.
```
Comment (cpoile) [inline source/collaborate/make-calls.rst:94]:
```suggestion
Lower a raised hand by accessing the host controls and selecting **Lower hand**. The participant will be notified that their hand was lowered by the host.
```

File: source/collaborate/make-calls.rst [hunk 5]
```diff
@@ -50,10 +56,47 @@ Start a call
...
+
 Share your screen
 -----------------
 
-During a call, you can share your screen with other call participants, unless your system admin has :ref:`disabled your ability to do so <configure/plugins-configuration-settings:allow screen sharing>`. 
+During a call, all call participants can share your screen with other call participants, unless your system admin has :ref:`disabled your ability to do so <configure/plugins-configuration-settings:allow screen sharing>`.
```
Comment (cpoile) [inline source/collaborate/make-calls.rst:99]: Hm, I think the original makes more sense, imo. We don't allow participants or hosts to share other people's screens. Unless you meant `all call participants can share their screen`?

**Review by cpoile — APPROVED** [review]
Awesome, thanks Carrie! Made some adjustments, please take or leave as you wish.

File: source/collaborate/make-calls.rst (hunk 5 above)
Comment (cwarnermm) [reply source/collaborate/make-calls.rst]: Great flag! I've corrected "your screen" to "their screen" for clarity.

**Pull request merged**